static/app/utils/metrics/mri.tsx

```diff
@@ -0,0 +1,93 @@
+import * as Sentry from '@sentry/react';
+
+import {t} from 'sentry/locale';
+import {MetricType, MRI, ParsedMRI} from 'sentry/types/metrics';
+import {parseFunction} from 'sentry/utils/discover/fields';
+
+import {DEFAULT_METRIC_ALERT_AGGREGATE, UseCase} from './index';
```

Comment:
should we move `DEFAULT_METRIC_ALERT_AGGREGATE` in here and rename it to `DEFAULT_METRIC_ALERT_FIELD`?